### DISCUSSION TITLE
scan for assets

### DISCUSSION THREAD

**Discussion by jvgassel** [minvws/nl-kat-coordination#21] [Q&A]
Would it be possible to use openkat to connect it to a customer network, and use it to sweep the network for alive hosts? So it would scan for example a /24 network with nmap and creates objects for all assets found.

**Comment by underdarknl** [comment]
Yes, this should be supported functionality.
Adding a netblock should be enough input for the nmap boefje to do its thing (currently it is not listening to netblocks), or it would be enough input for a BIT to generate all the related IpAddresses (this might create a mess in your graph though so we have not created this Bit yet).
Any found IP would then be input for Nmap again to do a more intensive scan and see if any useful ports are available for further testing.
